src/lib/registration/registration_ui.rb

```diff
@@ -181,8 +218,174 @@ def disable_update_repos(product_service)
       SwMgmt.set_repos_state(update_repos, false)
     end
 
+    def migration_products(products)
+      Yast::Popup.Feedback(
+        _(CONTACTING_MESSAGE),
+        _("Loading Migration Products...")) do
+        registration.migration_products(products)
+      end
+    end
+
+    # Register the selected addons, asks for reg. codes if required, known_reg_codes
+    # @param selected_addons [Array<Addon>] list of addons selected for registration,
+    #   successfully registered addons are removed from the list
+    # @param known_reg_codes [Hash] remembered reg. code, it's updated with the
+    #   user entered values
+    # @return [Symbol]
+    def register_addons(selected_addons, known_reg_codes)
+      # if registering only add-ons which do not need a reg. code (like SDK)
+      # then simply start the registration
+      if selected_addons.all?(&:free)
+        Yast::Wizard.SetContents(
+          # dialog title
+          _("Register Extensions and Modules"),
+          # display only the products which need a registration code
+          Empty(),
+          # help text
+          _("<p>Extensions and Modules are being registered.</p>"),
```

Comment:
html text should not be in the translation text. But anyway let it, if the translation already exists.